backend/node-postgres-promises/schema.sql

```diff
@@ -1,7 +1,7 @@
-DROP DATABASE IF EXISTS terrific_thrasher;
-CREATE DATABASE terrific_thrasher;
+DROP DATABASE IF EXISTS terrific_thrasher2;
```

Comment:
why do you need to delete the database everytime you run the schema? don't you just need to create the database once and delete the table and create the table?